docs/table_design/Data_distribution.md

```diff
@@ -183,41 +442,38 @@ DUPLICATE KEY(event_day,site_id,pv)
 DISTRIBUTED BY RANDOM BUCKETS 8; -- manually set the number of buckets to 8
 ```
 
-#### Hash bucketing
+### Hash bucketing
 
-Data in partitions can be subdivided into tablets based on the hash values of the bucketing columns and the number of buckets. In hash bucketing, the value of the bucketing column is used as input to calculate a hash value by using the hash function, and then the data is assigned to the corresponding bucket based on that hash value.
+StarRocks can use hash bucketing to subdivide data in a partition into buckets based on the bucketing key and [the number of buckets](#determine-the-number-of-buckets). In hash bucketing, a hash function takes data's bucketing key value as an input and calculates a hash value. Data is stored in the corresponding bucket based on the mapping between the hash values and buckets.
 
-##### Advantages
+**Advantages**:
 
-- Improved query performance: Rows with the same bucketing key value are assigned to the same bucket, which reduces the amount of data scanned during queries.
-- Even data distribution: By using the high-cardinality column (with a large number of unique values) as the bucketing key, data can be more evenly distributed across buckets.
+- Improved query performance: Rows with the same bucketing key values are stored in the same bucket, reducing the amount of data scanned during queries.
 
-##### Choose the bucketing key
+- Even data distribution: By selecting columns with higher cardinality (a larger number of unique values) as the bucketing key, data can be more evenly distributed across buckets.
 
-We recommend that you choose the column or columns that satisfy the following two requirements as the bucketing key.
+**How to choose the bucketing columns**
 
-- high cardinality column such as ID
-- column that often used as a filter in queries
+We recommend that you choose the column that satisfy the following two requirements as the bucketing column.
 
-But if the column that satisfies both requirements does not exist, you need to determine the bucketing column according to the complexity of queries.
+- high cardinality column such as ID
+- column that often used in a filter for queries
 
-- If the query is complex, it is recommended that you select the high cardinality column as the bucketing column to ensure that the data is as balanced as possible in each bucket and improve the cluster resource utilization.
-- If the query is relatively simple, then it is recommended to select the column that is often used as in the query condition as the bucketing column to improve the query efficiency.
+But if the column that satisfies both requirements does not exist, you need to determine the buckting column according to the complexity of queries.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/f7b6705e-dc8e-41d4-bb6a-6f8a154af43a)
use the plural form